src/transformers/models/cpmant/modeling_cpmant.py

```diff
@@ -659,7 +659,7 @@ def forward(
         output_hidden_states = (
             output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
         )
-        return_dict = return_dict if return_dict is not None else self.config.return_dict
+        return_dict = return_dict if return_dict is not None else self.config.use_return_dict
```

Comment:
this attribute is for torchscript, and we use this instead of `return_dict` everywhere